tests/positive/wsi.cpp

```diff
@@ -188,6 +188,10 @@ TEST_F(PositiveWsi, GetPhysicalDeviceSurfaceCapabilities2KHRWithFullscreenEXT) {
         GTEST_SKIP() << RequiredExtensionsNotSupported() << " not supported.";
     }
 
+    if (!IsPlatform(kMockICD)) {
+        GTEST_SKIP() << "Only run test MockICD due to CI stability";
+    }
+
```

Comment:
```suggestion
#ifndef VVL_TESTS_ENABLE_EXCLUSIVE_FULLSCREEN
    if (!IsPlatform(kMockICD)) {
        GTEST_SKIP() << "Only run test MockICD due to CI stability";
    }
#endif
```
So that running this test locally is simpler than having to remove this `if IsPlatform`?
Same for other test

Reply:
nothing, the dev has to explicitly supply it. I personally do so using the Visual Studio UI

Reply:
I agree with the intent. But not with the solution. That just solves the problem for fullscreen testing.

See `VkAmdBestPracticesLayerTest.TooManyPipelines` as an example.

We need a general purpose solution which would be better done in another PR.

When I see `VVL_TESTS_ENABLE_EXCLUSIVE_FULLSCREEN` I assume there is a CMake option to go along with it. Having to manually define the pre-processor define is non-obvious. Especially to contributors not familiar with our codebase.

Reply:
That's a good point
Then if a CMake option is ok for you, I am going to add it

Reply:
> Then if a CMake option is ok for you, I am going to add it

A CMake option would work. Ideally something like a CLI parameter or an environment variable.

As shown in this PR special build configurations like `LOADER_DISABLE_DYNAMIC_LIBRARY_UNLOADING` and `LOADER_USE_UNSAFE_FILE_SEARCH` make things more complicated.